### PR TITLE
chore: apply capitalization guidelines - iteration 8 (#12343)

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Apply capitalization guidelines defined in #12343.

@junlincc

### TEST PLAN
Check if modified texts are following capitalization guidelines.

### ADDITIONAL INFORMATION
- [x] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by junlincc** [comment]
please specify all your changes in this PR, include screenshot and test plan, also for all your future PRs. @michael-s-molina 🙏

**Comment by adam-stasiak** [comment]
I see error in one test which can be triggered by this change.
 Could you take care of this failing test? 
 AssertionError: Timed out retrying: Expected to find content: 'Add annotation layer' but never did.

File: superset-frontend/src/views/CRUD/csstemplates/CssTemplatesList.tsx [hunk 1]
```diff
@@ -246,7 +246,7 @@ function CssTemplatesList({
     subMenuButtons.push({
       name: (
         <>
-          <i className="fa fa-plus" /> {t('Css Template')}
+          <i className="fa fa-plus" /> {t('Css template')}
```
Comment (Steejay) [inline superset-frontend/src/views/CRUD/csstemplates/CssTemplatesList.tsx:249]: CSS is an acronym and should therefore be written as "CSS template"

**Review by Steejay — CHANGES_REQUESTED** [review]
Added a capitalization edit.

**Review by rusackas — CHANGES_REQUESTED** [review]
Thanks for the eagle eyes @Steejay! Once `CSS` is fixed, this looks good to go.

**Pull request merged**